app/frontend/src/i18n/formatters.tsx

```diff
@@ -0,0 +1,76 @@
+export default {
+    lowercase,
+    uppercase,
+    datetime,
+    number,
+    currency
+} as const;
```

Comment:
You have export in front of the functions and at the top, are both needed? (Maybe they are, I'm not a ES6 expert)

Reply:
The file is no longer needed, and I have deleted it.